lib/stat_tracker.rb

```diff
@@ -1,19 +1,24 @@
-require 'CSV'
```

Comment:
Just to clarify, stat_tracker does not need csv required?

Reply:
That was removed because we moved the CSV converting to the collections rather than the stattracker.